spec/factories/registration.rb

```diff
@@ -3,6 +3,7 @@
 FactoryGirl.define do
   factory :registration do
     user
-    conference
+    conference { create(:conference, registration_period: create(:registration_period, start_date: 3.days.ago)) }
```

Comment:
I would still like to see registration_period being created and properly defined in each test, and not in the factory.
Note: The factories of conference and registration_period currently have the right set of date ranges.